### PR TITLE
feat(networks): remove unused networks

### PR DESCRIPTION
**Pull request by Todmy**
Fixes https://github.com/snapshot-labs/pitches/issues/27

Changes proposed in this pull request:
- remove unused networks

Removed networks 

```
[
    "14",
    "16",
    "20",
    "31",
    "36",
    "44",
    "50",
    "51",
    "58",
    "65",
    "69",
    "70",
    "74",
    "87",
    "99",
    "106",
    "111",
    "119",
    "128",
    "188",
    "256",
    "288",
    "361",
    "499",
    "534",
    "595",
    "599",
    "647",
    "842",
    "940",
    "941",
    "1234",
    "1319",
    "1701",
    "1818",
    "1819",
    "2020",
    "2611",
    "4062",
    "4690",
    "5551",
    "5851",
    "7363",
    "7700",
    "9000",
    "9052",
    "11437",
    "12321",
    "12357",
    "22040",
    "32659",
    "42262",
    "43288",
    "60001",
    "70001",
    "70002",
    "70103",
    "71401",
    "71402",
    "333888",
    "333999",
    "666666",
    "888888",
    "900000",
    "278611351",
    "11297108109",
    "11297108099"
]

```

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -2437,37 +1263,6 @@
...
-    "network": "testnet",
-    "multicall": "0xcA11bde05977b3631167028862bE2a173976CA11",
-    "rpc": [],
-    "explorer": {
-      "url": "https://devnet.neonscan.org/"
-    },
-    "start": 205206112,
-    "logo": "ipfs://QmecRPQGa4bU7tybg1sUQY48Md9rWnmhrT6WW5ueqvhg6P",
-    "testnet": true
-  },
```
Comment (ChaituVR) [inline src/networks.json:2454]: This is added very recently 🙈 should be fine?

Comment (ChaituVR) [reply src/networks.json:2454]: https://github.com/snapshot-labs/snapshot.js/pull/897

File: src/networks.json [hunk 2]
```diff
@@ -397,72 +226,6 @@
...
-    "chainId": 75,
-    "network": "mainnet",
-    "multicall": "0x949d1A0757803C51F2EfFFEb5472C861A898B8E8",
-    "rpc": [],
-    "explorer": {
-      "url": "https://explorer.decimalchain.com"
-    },
-    "start": 16031065,
-    "logo": "ipfs://bafkreihkdhbce5rkogl63xegaarlirjrvbfarxbtbf5mqme3s5grvbjyxm"
-  },
```
Comment (ChaituVR) [inline src/networks.json:465]: This is added very recently, https://github.com/snapshot-labs/snapshot.js/pull/894 better to keep this

File: src/networks.json [hunk 3]
```diff
@@ -1253,23 +679,6 @@
...
-    "multicall": "0x751d21047C116413895c259f3f305e38C10B7cF6",
-    "rpc": [
-      "https://archive.evm.testnet.shimmer.network/v1/chains/rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3/evm"
-    ],
-    "explorer": {
-      "url": "https://explorer.evm.testnet.shimmer.network/"
-    },
-    "start": 10614,
-    "logo": "ipfs://QmYGxmEhV6djksUk97pdE9TmL6DXm9KW8BP7pwUv8pqp8r"
-  },
```
Comment (ChaituVR) [inline src/networks.json:1272]: added recently https://github.com/snapshot-labs/snapshot.js/pull/901

**Comment by Todmy** [comment]
reverted removal of networks 75, 204, 1072, 4337, 13337, 245022926 since they were added less than 2 months ago.

**Review by ChaituVR — APPROVED** [review]
tAck

**Pull request merged**